.github/dependabot.yml

```diff
@@ -19,5 +19,6 @@ updates:
       interval: "weekly"
       day: "friday"
       time: "06:00"
+      timezone: "Europe/Berlin"
```

Comment:
I'd suggest to also update the schedule of the `pip` package-ecosystem to match the `github-actions` schedule.

And, just out of experience: remove yourself as a reviewer in the configuration. Typically you are already notified when you get assigned to a PR or when a PR gets created in "your" repository. At least for me, this was sufficient in the past.

Reply:
Done.